tests/playwright-test/playwright.expect.misc.spec.ts

```diff
@@ -230,6 +230,36 @@ test('should support toHaveClass w/ array', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(1);
 });
 
+test('should support toContainClass', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.test.ts': `
+      const { test } = pwt;
+
+      test('pass', async ({ page }) => {
+        await page.setContent('<div class="foo bar baz"></div>');
+        const locator = page.locator('div');
+        await expect(locator).toContainClass('foo');
+        await expect(locator).toContainClass('bar');
+        await expect(locator).toContainClass('baz');
```

Comment:
lets make sure that we add test cases for:

"foo bar" (contains spaces)
and passing an array of strings: e.g. ["foobar", "baz"]

this should then cover all the cases which we want to support.

Reply:
done